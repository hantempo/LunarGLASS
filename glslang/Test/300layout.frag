#version 300 es
precision mediump float;
in vec4 pos;
layout (location = 2) in vec4 color;  // ERROR

layout(location = 1) out vec4 c;
layout(location = 3) out vec4 p;
layout(location = 4) out vec4 q[2];

void main()
{
    c = color;
    p = pos;
    q[1] = pos;
}
