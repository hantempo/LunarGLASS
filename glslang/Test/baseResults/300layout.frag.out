300layout.frag
ERROR: 0:4: 'location qualifier on input' : not supported in this stage: fragment
ERROR: 1 compilation errors.  No code generated.


ERROR: node is still EOpNull!
0:10  Function Definition: main( (void)
0:10    Function Parameters: 
0:12    Sequence
0:12      move second child to first child (mediump 4-component vector of float)
0:12        'c' (layout(location=1 ) out mediump 4-component vector of float)
0:12        'color' (layout(location=2 ) smooth in mediump 4-component vector of float)
0:13      move second child to first child (mediump 4-component vector of float)
0:13        'p' (layout(location=3 ) out mediump 4-component vector of float)
0:13        'pos' (smooth in mediump 4-component vector of float)
0:14      move second child to first child (mediump 4-component vector of float)
0:14        direct index (layout(location=4 ) mediump 4-component vector of float)
0:14          'q' (layout(location=4 ) out 2-element array of mediump 4-component vector of float)
0:14          Constant:
0:14            1 (const int)
0:14        'pos' (smooth in mediump 4-component vector of float)
0:?   Linker Objects
0:?     'pos' (smooth in mediump 4-component vector of float)
0:?     'color' (layout(location=2 ) smooth in mediump 4-component vector of float)
0:?     'c' (layout(location=1 ) out mediump 4-component vector of float)
0:?     'p' (layout(location=3 ) out mediump 4-component vector of float)
0:?     'q' (layout(location=4 ) out 2-element array of mediump 4-component vector of float)


Linked fragment stage:



